hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/SimpleRpcServer.java

```diff
@@ -68,6 +68,8 @@
  * while the server is up. CallRunner#run executes the call. When done, asks the included Call to
  * put itself on new queue for Responder to pull from and return result to client.
  * @see BlockingRpcClient
+ * @deprecated Since 2.4.14, 2.5.0, 2.6.0, will be removed in 4.0.0. Use {@link NettyRpcServer}
```

Comment:
I derived the versions from the following commits:
- https://github.com/apache/hbase/commit/9af341c672479f5521d9efd9ccc6b1c6c617a25e
- https://github.com/apache/hbase/commit/65057abd2ec3feb24781b8115f091e9d036e9ecb
- https://github.com/apache/hbase/commit/0018cbec583548ff447fb84465e7d52f2d1c4613
- https://github.com/apache/hbase/commit/ed32043da005475c7ae2d6e5e58f607592536213

Can you please check if the versions are correct?